packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -203,7 +203,7 @@ export default class LocusInfo extends EventsScope {
             file: 'locus-info',
             function: 'handleOneonOneEvent'
           },
-          EVENTS.REMOTE_ANSWERED_DECLINED,
```

Comment:
Can we leave the name as it is or change to something like REMOTE_RESPONSE or something 

REMOTE_ANSWERED would confuse some of us

Reply:
Can you also check if the remote side is receiving an event for decline